webui/react/package.json

```diff
@@ -101,6 +101,7 @@
     "@types/node": "^18.11.9",
     "@types/react": "^18.0.21",
     "@types/react-dom": "^18.0.8",
+    "@types/react-is": "^18.2.0",
```

Comment:
question: why not explicitly add `react-is`?

Reply:
i think we can use `react-is` without explicitly installing it because it's a dependency to other libraries, but we still need the type defs for our app code.